.github/workflows/create-snapcraft-pr-upon-release.yml

```diff
@@ -0,0 +1,30 @@
+name: Update Gradle version in snapcrafters/gradle repository
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+jobs:
+  Publish-PR-for-new-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout forked repository
+        uses: actions/checkout@v2
+        with:
+          repository: gradle/snapcrafters-gradle
+          token: ${{ secrets.PAM_FOR_CREATING_PR_AFTER_FINAL_PROMOTION }}
+      - name: Create pull request
+        env:
+          GITHUB_TOKEN: ${{ secrets.PAM_FOR_CREATING_PR_AFTER_FINAL_PROMOTION }}
+        run: |
+          version=$(echo ${GITHUB_REF#refs/*/} | tr -d v | sed 's/.0$//')
+          git remote add upstream https://github.com/snapcrafters/gradle
+          git fetch upstream master:upstream-master
+          git reset --hard upstream-master
+          git checkout -b "update-release/$version"
```

Comment:
I guess we can assume here that `$version` will always be a trusted input. Otherwise, I think that command injection _could_ be possible here?

Reply:
Oh, $version is from above, pulled from the `GITHUB_REF`, I don't think this is actually an issue